examples/demo/webpack.config.js

```diff
@@ -1,26 +1,14 @@
-import { createRequire } from 'module';
-
 import CssMinimizerPlugin from 'css-minimizer-webpack-plugin';
 import HtmlWebPackPlugin from 'html-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
-import webpack from 'webpack';
-
-const require = createRequire(import.meta.url);
 
 export default {
   // Output: {
   //   filename: '[contenthash].js',
   // },
-  externals: [{ 'vscode-emmet-helper-bundled': 'null' }],
   devtool: 'source-map',
   resolve: {
     extensions: ['.mjs', '.js', '.ts'],
-    fallback: {
```

Comment:
with the prebuild, we don't need the stubs in the demos anymore, so they were removed.

Reply:
Awesome!